targets/TARGET_Cypress/TARGET_PSOC6/TARGET_FUTURE_SEQUANA_M0_PSA/cyprotection.h

```diff
@@ -0,0 +1,189 @@
+/***************************************************************************//**
+* \file cyprotection.h
+* \version 1.0
+*
+* \brief
+*  This is the header file for the protection configuration structures
+*  and functions.
+*
+********************************************************************************
+* \copyright
+*
+* � 2018, Cypress Semiconductor Corporation
```

Comment:
What are this strange / special characters on Cypress license header? Are we ok with it?
CC @alzix

Reply:
Can this be cleaned (characters fixed) ?

Also, what license is this? Should be apache 2.0 or known permissive license. Other drivers from Cypress are Apache 2.0